File: openwhisk/msgaction/msgaction.py
def main(params):
    print ("Py msgaction params: " + str(params))

    for key in params["body"]:
        print(key, params["body"][key])

    return { "payload" :  "Hello " + str(params["body"]["text"]) + " from Python on OpenWhisk on OpenShift"}

File: openwhisk/msgaction/msgaction.js
function main(params) {
    console.log("JS msgaction args: %j", params);

    Object.keys(params["body"]).forEach(function (key) {
        console.log("%s: %j", key, params["body"][key])
    });

    return {payload: 'Hello ' + params.body.text + ' from JavaScript on OpenWhisk on Kubernetes '};
}
